{{cookiecutter.github_repository}}/settings/common.py

```diff
@@ -334,6 +335,16 @@
 CELERY_TIMEZONE = env('CELERY_TIMEZONE', default=TIME_ZONE)  # Use django's timezone by default
 {%- endif %}
 
+# USER AUTH - PASSWORD RESET
+# -----------------------------------------------------------------------------
+# see user.services.send_password_reset
+PASSWORD_RESET_CONFIRM_URL = env('PASSWORD_RESET_CONFIRM_URL',
+                                 default='reset-password/{token}/{uid}')
+
+# Api's typically run on a different name than the frontend app.
+FRONTEND_DOMAIN = env('FRONTEND_DOMAIN', default='http://localhost:8000')
```

Comment:
The variable to value mapping can be better. The value is a fully qualified url not just domain.